ios/BT/bridge/DebugMenuModule.m

```diff
@@ -18,7 +18,7 @@ @implementation DebugMenuModule
 
 RCT_EXPORT_METHOD(forceAppCrash)
 {
-  NSAssert(NO, @"Forced Crash (Debug)");
+  @throw [NSException exceptionWithName:NSGenericException reason:@"Forced Crash (Debug)" userInfo:nil];
```

Comment:
Should this one be part of the string extensions? Probably not since it's a debug one 😄